src/Models/UserModel.php

```
@@ -1,295 +1,325 @@
+    
+            if ($stmt->execute()) {
+                return "Gebruiker succesvol verwijdert.";
+            } else {
+                return "Fout bij het verwijderen van gebruiker met id: " . $id;
+            }            
+        } else {
+            return "Fout bij het verwijderen van de rollen voor gebruiker met id: " . $id;
+        }
+    }
+
+    public static function checkAccess($method, $requiredProperty, $userModel) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$method'. (undefined)

(UnusedFormalParameter)